### PR TITLE
Feature/sla footer

### PR DESCRIPTION
**Pull request by craigzour**
# Summary | Résumé

closes https://github.com/cds-snc/platform-forms-client/issues/1080

- Added new SLA pages (in both languages)
  - https://docs.google.com/document/d/1Uqu8lxiO9BrTnZOubJzSxe33q2RJKSdnTcnSXfr2mvA
  - https://docs.google.com/document/d/1P6qKkRIzHI7qqQa8SV1tXsgSRhpc0rKKzBp-iUDrFnk/edit
- Updated Footer component in order to display different set of links depending on provided props

# Test instructions | Instructions pour tester la modification

- You should see a footer with a "Terms and conditions" link (or Avis in French) on public form pages.
- You should see a footer with "Service level agreement" (or Accord sur les niveaux de service in French) and "Support" (or Soutien in French) links on pages that require authentication such as admin or form builder sections

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [x] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [x] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [x] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [x] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: public/static/locales/en/common.json [hunk 1]
```diff
@@ -3,6 +3,14 @@
     "terms": {
       "link": "/en/terms-avis",
       "desc": "Terms and conditions"
+    },
+    "sla": {
+      "link": "/en/sla",
```
Comment (thiessenp-cds) [inline public/static/locales/en/common.json:8]: Adding logic like links in a translation may confuse people. Though we do this in other files like error.json. So I guess it's consistent but just noting for the future :)

File: lib/routeUtils.ts [hunk 2]
```diff
@@ -20,11 +20,3 @@ export const getPageNameUrl = (): string => {
 export const isSplashPage = (): boolean => {
   return getPageNameUrl() === "splash";
 };
-
```
Comment (thiessenp-cds) [inline lib/routeUtils.ts:23]: Just to make sure, this function is being removed because it is not being used in the code base?

**Review by thiessenp-cds — APPROVED** [review]
Some minor comments but LGTM :)

**Review by Moro-Code — APPROVED** [review]
Looks good to me

**Pull request merged**